### PR TITLE
chore(celery): Change local Celery Pycharm run to solo for better debugging

### PR DESCRIPTION
**Pull request by webjunkie**
## Problem

Python code in Celery was hard to debug because it spawned subprocesses/workers.

## Changes

- run as `solo` locally, so it runs single threaded
- works great with Pycharm debugger now
- just still remember to restart when making code changes

## How did you test this code?

- tested locally

### CODE REVIEW

**Review by pauldambra — APPROVED** [review]
:shipit:

File: .run/Celery.run.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
     <option name="ADD_SOURCE_ROOTS" value="true" />
     <EXTENSION ID="PythonCoverageRunConfigurationExtension" runner="coverage.py" />
     <option name="SCRIPT_NAME" value="$PROJECT_DIR$/env/bin/celery" />
-    <option name="PARAMETERS" value="-A posthog worker -B --scheduler redbeat.RedBeatScheduler --without-heartbeat --without-mingle --loglevel=DEBUG" />
+    <option name="PARAMETERS" value="-A posthog worker -B --scheduler redbeat.RedBeatScheduler --without-heartbeat --without-mingle --loglevel=DEBUG --pool=solo" />
```
Comment (pauldambra) [inline .run/Celery.run.xml:23]: do we need a similar change anywhere else?

bin/start?

vscode run?

Comment (webjunkie) [reply .run/Celery.run.xml:23]: How do other people run it? 😅 

I can check...

Comment (webjunkie) [reply .run/Celery.run.xml:23]: I adjusted vscode and bin/start (hope the latter is not used somewhere non-local)

Comment (pauldambra) [reply .run/Celery.run.xml:23]: ooh... ah, I remember breaking prod by assuming the `bin` folder was local... so at least for one command it's not only local!

Comment (pauldambra) [reply .run/Celery.run.xml:23]: let's only do vscode and pycharm now I think about it

**Pull request merged**